torch/fx/experimental/symbolic_shapes.py

```diff
@@ -5771,6 +5772,25 @@ def simplify(self, expr: _SympyT) -> _SympyT:
         """Use known constraints and replacements to simplify the given expr"""
         expr = safe_expand(expr)
         expr = self.replace(expr)
+
+        if expr.has(Max):
+            max_replacements = {}
+            for atom in expr.atoms(Max):
+                a, b = atom.args
```

Comment:
i mean what about
torch._check(u0 + u1 + u2 + u3+ u4 + u5 == 20)
if guard_size_oblivious(torch.sym_max(6, u0 + u1 + u2 + u3+ u4 + u5+..) == 20):

this feels like a fix for a specific use case, I do am not sure how i feel about it.